README.md

```diff
@@ -36,3 +36,20 @@ There is also support for different workflows to trigger sync of the labels with
 ```
 
 Because GitHub API doesn't provide information about linked issues this workflow will parse the information from the `body` of the issue.
+
+## Custom keywords
+
+You can provide custom keywords which will be picked up by the parser.
+
+These keywords will not override github specified keywords.
```

Comment:
Are the github specified keywords automatically picked up by the parser ? I think not, right ?

Reply:
Yes, they are.. by this update, they cannot be overridden. They will always be picked up.